cirq-google/cirq_google/devices/xmon_device_test.py

```diff
@@ -43,7 +43,8 @@ def qubits(self):
         raise NotImplementedError()
 
 
-def test_init():
+@mock.patch.dict(os.environ, clear='CIRQ_TESTING')
```

Comment:
You can just add a `ALLOW_DEPRECATION_IN_TEST = 'ALLOW_DEPRECATION_IN_TEST'` at the beginning of the file instead of adding a patch to each individual test

Reply:
Try as I might I couldn't get this for some reason or other, so I just left as mocks.